tests/usbredir_test.go

```diff
@@ -175,10 +175,27 @@ var _ = Describe("[crit:medium][vendor:cnv-qe@redhat.com][level:component][sig-c
 	})
 })
 
-func createVMI(virtClient kubecli.KubevirtClient, enableUsbredir bool) (*v1.VirtualMachineInstance, error) {
-	randomVmi := tests.NewRandomVMI()
-	if enableUsbredir {
-		randomVmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
+func withClientPassthrough() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		vmi.Spec.Domain.Devices.ClientPassthrough = &v1.ClientPassthroughDevices{}
 	}
-	return virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVmi)
+}
+
+func createVMIWithUsbredirEnabled(virtClient kubecli.KubevirtClient) *v1.VirtualMachineInstance {
+	randomVmi := libvmi.New(
+		libvmi.WithResourceMemory("32Mi"),
+		withClientPassthrough(),
+	)
+	vmi, err := virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(randomVmi)
```

Comment:
using `tests.RunVMIAndExpectLaunch()` would remove much of this boilerplate.

Reply:
Yes but don't we want to delete tests/utils.go ?
This will increase the coupling to this file.